src/Helpers/Consign/index.ts

```diff
@@ -0,0 +1,113 @@
+/**
+ * Helpers for /consign app.
+ *
+ * @see https://docs.google.com/spreadsheets/d/12wj1rt-j5SguboPJMuaM1GqJD-5B5u5piipw8MQ5Zuk/edit#gid=0 for matrix.
+ */
+
+import {
+  ActionType,
+  ClickedAppDownload,
+  ClickedArticleGroup,
+  ClickedArtworkGroup,
+  ContextModule,
+  OwnerType,
+} from "../../Schema"
+import {
+  FocusedOnSearchInput,
+  SearchedWithNoResults,
+  SelectedItemFromSearch,
+} from "../../Schema/Events/Search"
+
+type ExcludeDefined<T> = Required<
+  Omit<
+    T,
+    "action" | "context_module" | "context_owner_type" | "type" | "subject"
+  >
+>
+
+export const focusedOnSearchInput = (
+  props: ExcludeDefined<FocusedOnSearchInput>,
+): FocusedOnSearchInput => {
+  return {
+    action: ActionType.focusedOnSearchInput,
+    context_module: ContextModule.priceEstimate,
+    context_owner_id: props.context_owner_id,
+    context_owner_slug: props.context_owner_slug,
+    context_owner_type: OwnerType.consign,
+  }
+}
+
+export const selectedItemFromSearch = (
+  props: ExcludeDefined<SelectedItemFromSearch>,
+): SelectedItemFromSearch => {
+  return {
+    action: ActionType.selectedItemFromSearch,
+    context_module: ContextModule.priceEstimate,
+    context_owner_id: props.context_owner_id,
+    context_owner_slug: props.context_owner_slug,
+    context_owner_type: OwnerType.consign,
+    destination_owner_id: props.destination_owner_id,
+    destination_owner_slug: props.destination_owner_slug,
+    destination_owner_type: props.destination_owner_type,
+    owner_id: props.owner_id,
+    owner_slug: props.owner_slug,
+    owner_type: props.owner_type,
+    query: props.query,
+  }
+}
+
+export const searchedWithNoResults = (
+  props: ExcludeDefined<SearchedWithNoResults>,
+): SearchedWithNoResults => {
+  return {
+    action: ActionType.searchedWithNoResults,
+    context_module: ContextModule.priceEstimate,
+    context_owner_id: props.context_owner_id,
+    context_owner_slug: props.context_owner_slug,
+    context_owner_type: OwnerType.consign,
+    destination_owner_type: props.destination_owner_type,
+    query: props.query,
+  }
+}
+
+export const clickedArtworkGroup = (
+  props: ExcludeDefined<ClickedArtworkGroup>,
+): ClickedArtworkGroup => {
+  return {
+    action: ActionType.clickedArtworkGroup,
+    context_module: ContextModule.artworkRecentlySoldGrid,
+    context_page_owner_type: OwnerType.consign,
+    destination_page_owner_id: props.destination_page_owner_id,
+    destination_page_owner_slug: props.destination_page_owner_slug,
+    destination_page_owner_type: props.destination_page_owner_type,
+    type: "thumbnail",
+  }
+}
+
+export const clickedArticleGroup = (
```

Comment:
This is a click event that is not specific to consign contexts and should live in the `Click` folder: https://github.com/artsy/cohesion/tree/master/src/Helpers/Click